rules/3-steps-to-a-pbi/rule.md

```diff
@@ -27,39 +27,36 @@ From a developer's viewpoint, the lifecycle of a PBI can be broken down into 3 s
 
 ### Before it's ready
 
-Got an email with tasks or bugs? [Add them to the Product Backlog as PBIs]((/turn-emails-into-pbis) as soon as possible!
-
-Next, check the PBI against the [Definition of Ready](/have-a-definition-of-ready).
+Got an email with tasks or bugs? [Add them to the Product Backlog as PBIs](/turn-emails-into-pbis) as soon as possible!
 
 ### 1. PBI is Ready
 
 You have a backlog of PBIs (tasks, features, and fixes) which are already prioritized and approved by the [Product Owner](/rules-to-better-product-owners)
 
-1. Take the next PBI in the list by assigning it to yourself
-2. Break down that PBI into clear and actionable tasks
+1. Take the next PBI in the list and check it against the Definition of Ready
+2. Assign the PBI to yourself
+3. Break down that PBI into clear and actionable tasks
 
 ![Figure: "Testing Task" added to a PBI. This is the board the team will use for 90% of the Sprint, so that testing is visible to everyone](Testing-task.png)
```

Comment:
```suggestion
![Figure: "Testing Task" added to a PBI. This is the board the team will use for 90% of the Sprint, so that testing is visible to everyone](testing-task.png)
```
as per https://www.ssw.com.au/rules/how-to-name-documents/